gitlab-runner.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: gitlab-runner
+  version: 16.2.0
+  epoch: 1
+  description: GitLab Runner is the open source project that is used to run your CI/CD jobs and send the results back to GitLab
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - busybox
+      - go
+  environment:
+    CGO_ENABLED: "1"
+    GOEXPERIMENT: boringcrypto
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://gitlab.com/gitlab-org/gitlab-runner
+      tag: v${{package.version}}
+      expected-commit: 782e15dae28e70213e21b29f2fed257d2cf2998c
+
+  - uses: go/build
+    with:
+      packages: .
+      tags: fips
```

Comment:
To stay consistent with other fipsey packages, we should have separate `gitlab-runner` and `gitlab-runner-fips` packages. You can do this with subpackages too.

Reply:
@imjasonh and do you generally only include `openssl` as a runtime dependency on the fipsy variants?

Reply:
Answered above: https://github.com/wolfi-dev/os/pull/4202#discussion_r1284749997